pkg/controller/clusterprovision/clusterprovision_controller.go

```diff
@@ -285,6 +288,33 @@ func (r *ReconcileClusterProvision) reconcileRunningJob(instance *hivev1.Cluster
 
 	pLog.Debug("install job still running")
 
+	if time.Since(job.CreationTimestamp.Time) > podStatusCheckDelay {
+		installPod, err := r.getInstallPod(job, pLog)
+		if err != nil {
+			pLog.WithError(err).Error("could not get install pod")
+			if err := r.setCondition(instance, hivev1.InstallPodStuckCondition, corev1.ConditionTrue, "InstallPodMissing", err.Error(), controllerutils.UpdateConditionIfReasonOrMessageChange, pLog); err != nil {
+				return reconcile.Result{}, err
+			}
+			return reconcile.Result{}, err
```

Comment:
@akhil-rane @staebler does it matter that the provision instance won't be requeued after this?

Reply:
Why would it not be requeued? The reconcile is returning a non-nil error.

Reply:
Ahh you're right, for whatever reason I was thinking it'd be nil if the previous stmt was successful, but yeah, it's only scoped to that stmt